pkg/controllers/machine/lifecycle/launch.go

```diff
@@ -88,7 +92,12 @@ func (l *Launch) linkMachine(ctx context.Context, machine *v1alpha5.Machine) (*v
 		}).Inc()
 		return nil, nil
 	}
-	logging.FromContext(ctx).Debugf("linked machine")
+	logging.FromContext(ctx).With(
+		"provider-id", created.Status.ProviderID,
+		"instance-type", created.Labels[v1.LabelInstanceTypeStable],
+		"zone", created.Labels[v1.LabelTopologyZone],
+		"capacity-type", created.Labels[v1alpha5.LabelCapacityType],
+		"capacity", created.Status.Capacity).Infof("linked machine")
```

Comment:
Do we care more about capacity than allocatable? I think it's more useful to the user to see allocatable, but I could go either way.

Reply:
You're probably right, allocatable is more useful here